src/node/rpc/node_frontend.h

```diff
@@ -1394,10 +1396,13 @@ namespace ccf
         auto js_engine_options = js_engine_map->get();
         m.max_stack_size = 1024 * 1024;
         m.max_heap_size = 100 * 1024 * 1024;
+        m.max_execution_time = 240;
```

Comment:
These defaults should be defined somewhere else I think, with the same values as `wrap.cpp:110-112`.

Reply:
Do you mean they shouldn't be hardcoded for the `js_metrics` response? (although yes this value should be updated from 240 to 1000 to reflect latest testing). IMO

Reply:
I think there should be a single source of truth for the default values for those, otherwise as you said, they could drift and we could end up reporting incorrect values in the endpoint. Perhaps they could move to `wrap.h` that we could include from `node_frontend.h` too?

Reply:
Yup that makes sense